src/v/kafka/server/handlers/alter_configs.cc

```diff
@@ -70,23 +70,55 @@ create_topic_properties_update(
      * factor, if not set in the request explicitly it will not be overriden.
      */
     update.properties.compaction_strategy.op
-      = cluster::incremental_update_operation::set;
+      = cluster::incremental_update_operation::remove;
     update.properties.compression.op
-      = cluster::incremental_update_operation::set;
+      = cluster::incremental_update_operation::remove;
     update.properties.segment_size.op
-      = cluster::incremental_update_operation::set;
+      = cluster::incremental_update_operation::remove;
     update.properties.timestamp_type.op
-      = cluster::incremental_update_operation::set;
+      = cluster::incremental_update_operation::remove;
     update.properties.retention_bytes.op
-      = cluster::incremental_update_operation::set;
-    update.properties.retention_duration.op
-      = cluster::incremental_update_operation::set;
+      = cluster::incremental_update_operation::remove;
+    update.properties.remote_delete.op
+      = cluster::incremental_update_operation::remove;
+    update.properties.segment_ms.op
+      = cluster::incremental_update_operation::remove;
     update.properties.shadow_indexing.op
-      = cluster::incremental_update_operation::set;
-    update.custom_properties.replication_factor.op
-      = cluster::incremental_update_operation::none;
-    update.custom_properties.data_policy.op
-      = cluster::incremental_update_operation::none;
+      = cluster::incremental_update_operation::remove;
```

Comment:
this could probably be formatted better if you used a type alias for the enum class like
```
using op_type = cluster::increemtnal_update_operation;
update.properties.batch_max_bytes.op = op_type::remove;
```

Reply:
Yes that's nicer. Done